README.md

```diff
@@ -13,7 +13,7 @@ _Components are designed for deployment in adversarial environments. If you find
 * [client](./client/README.md): Client for interacting with `alto`.
 * [explorer](./explorer/README.md): Visualize `alto` activity.
 * [inspector](./inspector/README.md): Inspect `alto` activity.
-* [indexer](./indexer/README.md): Serve `alto` activity locally.
+* [indexer](./indexer/README.md): Serve `alto` activity.
```

Comment:
I added `locally` intentionally since this is by no means an indexer that could be used in any other context than development, since it has unbounded memory requirements.